transformers/test_data/vow_file.go

```diff
@@ -48,13 +48,13 @@ var rawVowFileLog = types.Log{
 var VowFileEventLog = core.EventLog{
 	ID:          int64(rand.Int31()),
 	HeaderID:    int64(rand.Int31()),
-	Log:         rawVowFileLog,
+	Log:         rawVowFileAuctionAttributesLog,
 	Transformed: false,
 }
 
 var VowFileModel = event.InsertionModel{
```

Comment:
Same here - also wondering if we might want to use `CopyModel` to hedge against test pollution

Reply:
Cool. I noticed this and got these in #251